code/game/gamemodes/wizard/artefact.dm

```diff
@@ -633,11 +633,6 @@ var/global/list/multiverse = list()
 	if(M.stat != DEAD)
 		to_chat(user, "<span class='warning'>This artifact can only affect the dead!</span>")
 		return
-
-	if(!M.mind || !M.client)
```

Comment:
This should probably stick around; the mind isn't destroyed by ghosting, and if they don't have a client because they quit, we don't want them wasting a charge.